lib/managers/files.js

```diff
@@ -575,6 +575,73 @@ Files.prototype.deletePermanently = function(fileID, callback) {
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+
+ /**
+  * Enum of valid actions for locking and unlocking files
+  *
+  * @enum {LockType}
+  */
+Files.prototype.lockType = {
```

Comment:
Do we need to expose these in particular? What could be the use case out side of lock and unlock functions?